web/packages/teleport/src/Login/useLogin.ts

```diff
@@ -134,7 +134,12 @@ type LoginResponse = {
 };
 
 function authorizeWithDeviceTrust(token: DeviceWebToken) {
-  const authorize = cfg.getDeviceTrustAuthorizeRoute(token.id, token.token);
+  let redirect = history.getRedirectParam();
```

Comment:
I don't like the fact that  we pass around the whole URL, even though we end up using only the path, but I guess it's a tech debt from how the Web UI does app redirects, right?

Reply:
Yeah this is just how its passed around unfortunately. I can add a TODO to fix later and inspect how many spots it will touch